src/interactivity/win32/precomp.h

```diff
@@ -1,4 +1,10 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+#pragma prefast(push)
+#pragma prefast(disable : 26071, "Range violation in Intsafe. Not ours.")
+#define ENABLE_INTSAFE_SIGNED_FUNCTIONS // Only unsigned intsafe math/casts available without this def
+#include <intsafe.h>

```

Comment:
I remember that interactivity should not be changed in this repo. But I removed the `<intsafe.h>` in `../../host/precomp.h`  so I had to add it back here to make the project compile.